.ci/component_descriptor

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env bash
 
+chmod +x "$(dirname $0)"/../vendor/github.com/gardener/gardener/hack/.ci/component_descriptor
```

Comment:
This will not really help. I get your point but today `.ci/check` only calls `make check` and `make check-generate`. It does not call `make revendor`. So even if all permissions are moved there it will not work. What i can do is create another target which only sets the permissions on scripts and make that as a pre-requisite for make targets.

Reply:
I have done the changes, PTAL

Reply:
Thanks